### PR TITLE
Add limit metrics.

### PR DESCRIPTION
**Pull request by PhilippKilian**

### CODE REVIEW

File: b3lb/rest/b3lb/tasks.py [hunk 1]
```diff
@@ -28,6 +28,11 @@
 import json
 from jinja2 import Template
 
+#
+# Celery task routines
+#
+LIMIT_METRICS = [("attendee_limit", "Attendee soft limit"), ("meeting_limit", "Attendee soft limit")]
```
Comment (liske) [inline b3lb/rest/b3lb/tasks.py:34]: meeting_limit:  `Meeting limit`

File: b3lb/rest/templates/metrics_all [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
 # TYPE b3lb_node_load gauge
 {% for node, cluster, load in nodes %}b3lb_node_load{node="{{ node }}",cluster="{{ cluster }}"} {{ load }}
-{% endfor %}# HELP secret_endpoint_limit Attendee limit per secret
-# TYPE secret_endpoint_limit gauge
-{% for secret in secrets %}b3lb_endpoint_limit{secret="{{ secret }}"} 0
-{% endfor %}{% for metric_key, metric_value in metrics.items() %}# HELP b3lb_{{ metric_key }} {{ metric_value.name_choice }}
+{% endfor %}{% if tenant_limits %}# HELP b3lb_secret_tenant_limit Attendee limit per tenant
+# TYPE b3lb_tenant_attendee_limit gauge
+{% for slug, attendee, meeting in tenant_limits %}b3lb_tenant_limit_attendee{tenant="{{ slug }}"} {{ attendee }}
+{% endfor %}# HELP b3lb_tenant_limit_meeting Meeting limit per tenant
```
Comment (liske) [inline b3lb/rest/templates/metrics_all:7]: Should look like:
```python
# TYPE foo
# HELP bar
metrci1....
```
w/o any newlines

File: b3lb/rest/templates/metrics_all [hunk 3]
```diff
@@ -1,10 +1,19 @@
 # HELP b3lb_node_load Calculated node load
 # TYPE b3lb_node_load gauge
 {% for node, cluster, load in nodes %}b3lb_node_load{node="{{ node }}",cluster="{{ cluster }}"} {{ load }}
-{% endfor %}# HELP secret_endpoint_limit Attendee limit per secret
-# TYPE secret_endpoint_limit gauge
-{% for secret in secrets %}b3lb_endpoint_limit{secret="{{ secret }}"} 0
-{% endfor %}{% for metric_key, metric_value in metrics.items() %}# HELP b3lb_{{ metric_key }} {{ metric_value.name_choice }}
+{% endfor %}{% if tenant_limits %}# HELP b3lb_secret_tenant_limit Attendee limit per tenant
```
Comment (liske) [inline b3lb/rest/templates/metrics_all:4]: Metrics naming suggestion:
- `b3lb_secret_attendee_limit{secret=...}`
- `b3lb_secret_meeting_limit{secret=...}`
- `b3lb_tenant_attendee_limit{secret=...}`
- `b3lb_tenant_meeting_limit{secret=...}`

`TYPE` hint is missing?

File: b3lb/rest/templates/metrics_secret [hunk 4]
```diff
@@ -1,7 +1,16 @@
-# HELP b3lb_secret_limit Attendee limit per secret
-# TYPE b3lb_secret_limit gauge
-{% for secret in secrets %}b3lb_secret_limit{secret="{{ secret }}"} 0
-{% endfor %}{% for metric_key, metric_value in metrics.items() %}# HELP b3lb_{{ metric_key }} {{ metric_value.name_choice }}
+{% if tenant_limits %}# HELP b3lb_secret_tenant_limit Attendee limit per tenant
+# TYPE b3lb_tenant_attendee_limit gauge
+{% for slug, attendee, meeting in tenant_limits %}b3lb_tenant_limit_attendee{tenant="{{ slug }}"} {{ attendee }}
+{% endfor %}# HELP b3lb_tenant_limit_meeting Meeting limit per tenant
```
Comment (liske) [inline b3lb/rest/templates/metrics_secret:4]: Same as in `metrics_all`

**Review by liske — CHANGES_REQUESTED** [review]

File: b3lb/rest/b3lb/tasks.py (hunk 1 above)
Comment (PhilippKilian) [reply b3lb/rest/b3lb/tasks.py:34]: It's an artifact from the development and wasn't used.

File: b3lb/rest/templates/metrics_all (hunk 3 above)
Comment (PhilippKilian) [reply b3lb/rest/templates/metrics_all:4]: Renamed metrics. `TYPE` already exists.

File: b3lb/rest/templates/metrics_secret (hunk 4 above)
Comment (PhilippKilian) [reply b3lb/rest/templates/metrics_secret]: done

File: b3lb/rest/templates/metrics_all (hunk 2 above)
Comment (PhilippKilian) [reply b3lb/rest/templates/metrics_all]: nothing changed, looks fine

File: b3lb/rest/templates/metrics_all [hunk 5]
```diff
@@ -1,10 +1,19 @@
 # HELP b3lb_node_load Calculated node load
 # TYPE b3lb_node_load gauge
 {% for node, cluster, load in nodes %}b3lb_node_load{node="{{ node }}",cluster="{{ cluster }}"} {{ load }}
-{% endfor %}# HELP secret_endpoint_limit Attendee limit per secret
-# TYPE secret_endpoint_limit gauge
-{% for secret in secrets %}b3lb_endpoint_limit{secret="{{ secret }}"} 0
-{% endfor %}{% for metric_key, metric_value in metrics.items() %}# HELP b3lb_{{ metric_key }} {{ metric_value.name_choice }}
+{% endfor %}{% if tenant_limits %}# HELP b3lb_tenant_attendee_limit Attendee limit per tenant
```
Comment (liske) [inline b3lb/rest/templates/metrics_all:4]: Isn't `tenant_limits` and `secret_limits` always evaluated to true? So the conditionals should be dropped, shouldn't it?

**Review by liske — CHANGES_REQUESTED** [review]

File: b3lb/rest/templates/metrics_all (hunk 5 above)
Comment (PhilippKilian) [reply b3lb/rest/templates/metrics_all]: if statements dropped.

**Review by liske — APPROVED** [review]

**Pull request merged**